test/Selenium/.classpath

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Normally IDE specific files (.classpath, .project, .settings) are left out of source control since they can be specific to the Eclipse installation. That being said, I'm not too bothered either way.